src/bin/zmrabbitmqctl

```diff
@@ -0,0 +1,174 @@
+#!/bin/sh
+# 
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2016, 2021 Synacor, Inc.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+# 
+NAME=rabbitmq-server
+DAEMON=/opt/zimbra/common/sbin/${NAME}
+CONTROL=/opt/zimbra/common/sbin/rabbitmqctl
+DESC=rabbitmq
+USER=zimbra
+PID_FILE=/opt/zimbra/data/rabbitmq/pid
+LOCK_FILE=/opt/zimbra/data/rabbitmq/$NAME
+RABBITMQ_SERVER_CONSOLE_OUTPUT_DIR=/opt/zimbra/log/rabbitmq
```

Comment:
not sure if we need folder for rabbitmq
we can use /opt/zimbra/log as path and rename files to rabbitmq.log and rabbitmq_err.log

Reply:
hmm there is another log file also https://github.com/Zimbra/packages/pull/169/files#diff-efca594a6905f9186d565e38fc00aa5078c5fa6add7085d27952b7e9cfc33490R6
a rethought, we can have one log file rabbitmq.log which will be saving from https://github.com/Zimbra/packages/pull/169/files#diff-efca594a6905f9186d565e38fc00aa5078c5fa6add7085d27952b7e9cfc33490R6
and another rabbitmq_startup.log and we can pipe stdout and stderr both outputs to this file, as logs coming from that command will be very less

Reply:
committed changes for same